docs/v3/source/includes/resources/manifests/_object.md.erb

```diff
@@ -89,7 +89,8 @@ This configuration is specified per application and applies to all of the applic
 Name | Type | Description
 ----- | ---- | -----------
 **name** | _string_ | Name of the app
-**buildpacks** | _list of strings_ | a) An empty array, which will automatically select the appropriate default buildpack according to the coding language <br>b) An array of one or more URLs pointing to buildpacks <br>c) An array of one or more installed buildpack names <br>Replaces the legacy `buildpack` field
+**lifecycle** | _string_ | Type of the lifecycle; valid values are `buildpack`, `cnb`, `docker`.
```

Comment:
Should add the defaults:
Defaults to `docker` if the manifest contains docker configuration, otherwise the default is `buildpack`.